KarhooUISDK/Common/KarhooTextInputView/KarhooTextInputView.swift

```diff
@@ -195,7 +195,7 @@ class KarhooTextInputView: UIView, KarhooInputView {
         }
 
         textView.text = value
-        textView.textColor = KarhooUI.colors.guestCheckoutDarkGrey
+        textView.textColor = KarhooUI.colors.infoColor
```

Comment:
quick note from my side, the infoColor was defined under this name because the requirement in the story was to have “black” as a defined color which was not explicit enough, therefore Cornel and I talked about naming it like this so it makes more sense.